src/Cli/dotnet/commands/dotnet-run/LaunchSettings/LaunchSettingsManager.cs

```diff
@@ -40,12 +41,13 @@ public static LaunchSettingsApplyResult TryApplyLaunchSettings(string launchSett
                         return new LaunchSettingsApplyResult(false, LocalizableStrings.LaunchProfilesCollectionIsNotAJsonObject);
                     }
 
+                    var selectedProfileName = profileName;
```

Comment:
nit: if this is nullable, lean towards using nullable enable and marking the type with ?. I don't think profile name does it either, but really, it should...

Reply:
Done.